instrumentation/opentelemetry/init.go

```diff
@@ -101,6 +113,36 @@ func makeExporterFactory(cfg *config.AgentConfig) func() (sdktrace.SpanExporter,
 	}
 }
 
+func createTLSConfig(cfg *config.AgentConfig) *tls.Config {
```

Comment:
If following the comment below we can also narrow down this to `*config.Reporting`.

Reply:
done